livekit_agent.proto

```diff
@@ -74,38 +80,72 @@ enum JobStatus {
     JS_FAILED = 2;
 }
 
+message SimulateJobRequest {
+    JobType type = 1;
+    string room_id = 2;
+    string participant_id = 3;
+}
+
+message WorkerPing {
+    int64 timestamp = 1;
+}
+message WorkerPong {
+    int64 last_timestamp = 1;
+    int64 timestamp = 2;
+}
+
 message RegisterWorkerRequest {
     JobType type = 1;
     string worker_id = 2;
     string version = 3;
     string name = 4;
+    uint32 ping_interval = 5;
+    optional string namespace = 6;
 }
 
 message RegisterWorkerResponse {
     string worker_id = 1;
     string server_version = 2;
 }
 
+message MigrateJobRequest {
+    Job job = 1;
```

Comment:
should this just be a job_id? if server has knowledge of all of the jobs, then it seems that just a job id is sufficient.

Reply:
yeah sgtm